### DISCUSSION TITLE
Github Pages site detected as malicious

### DISCUSSION THREAD

**Discussion by Impre-visible** [community/community#46444] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, I have a simple github pages for the documentation of my app. I have to github pages, on at chocolateapp.github.io and another one at chocolateapp.github.io/doc.
The one a /doc is detected as a malicious website. Maybe because I use retype ? Not sure why I have this error, but it's not good, I contacted Google Chrome for that, and reported a false positive. If someone can help me, it will be great.

**Comment by SilasRodrigues19** [comment]
Hey @Impre-visible 

I searched here in the browser and it's normal, it shows that the connection is secure.

![image](https://user-images.githubusercontent.com/47532703/216841001-7d33028a-0aa9-4530-810e-6cb99bbb892d.png)

**Reply by Impre-visible** [reply]
Yeah, I checked with another browser and it's good, don't know why. Maybe a bug ? Hope it's stay like that.

**Reply by SilasRodrigues19** [reply]
I found an answer in that thread here:

#40244
